containers/images/pulp/container-assets/route_paths.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import pkgutil
+import json
+import re
+import sys
+import nginx
+from pathlib import Path
+
+nginx_configs = []
+
+for i in pkgutil.iter_modules():
+    # filter pulp and galaxy packages
+    if re.search(r'pulp_.*|galaxy_ng',i.name):
+        # only add the filename to array if it exists
+        if Path(i.module_finder.path + "/" + i.name + "/app/webserver_snippets/nginx.conf").is_file():
+            nginx_configs.append(i.module_finder.path + "/" + i.name + "/app/webserver_snippets/nginx.conf")
```

Comment:
nice!

Reply:
don't you want to extract this into a variable? `i.module_finder.path + "/" + i.name + "/app/webserver_snippets/nginx.conf"`

Reply:
https://github.com/pulp/pulp-operator/pull/494/files#diff-3b5362d853b2987b11361bef8477c7dd984fba422a247b47166b7879d18d69dcR15-R18